react/withCssHandles.tsx

```diff
@@ -16,7 +17,9 @@ const withCssHandles = <T extends CssHandlesInput, ComponentProps>(
 
   const displayName = Component.displayName || Component.name || 'Component'
   EnhancedComponent.displayName = `withCssHandles(${displayName})`
-
+  hoistNonReactStatic(EnhancedComponent, Component)
```

Comment:
I think this should be done
```suggestion
  return hoistNonReactStatic(EnhancedComponent, Component)
```

Reply:
I might be mistaken though, please confirm